lib/at_cmd_parser/at_utils.h

```diff
@@ -242,6 +242,30 @@ static inline bool is_command(const char *str)
 	return false;
 }
 
+/**
+ * @brief Check if a string is a beginning of an AT CLAC response
+ *
+ * This function will check if the string is a CLAC response prefix.
+ * Valid prefixes: AT+ and AT%, except AT%X
+ * @param[in] str String to examine
```

Comment:
Let's add an empty line above, currently, the param list blends with the description.

Reply:
Done